app/src/main/res/layout/contactpatient_callpatient.xml

```diff
@@ -0,0 +1,347 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Here we are adding a new layout instead of renaming like the commit message mentions, can you rename this, please?

Reply:
TBH, we can combine this with `Implement CallPatientView` commit